src/Evolution/Systems/ScalarWave/Characteristics.hpp

```diff
@@ -0,0 +1,225 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/FaceNormal.hpp"
+#include "Evolution/Systems/ScalarWave/Tags.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/ForceInline.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+template <typename>
+class Variables;
+
+class DataVector;
+
+namespace Tags {
+template <typename Tag>
+struct Normalized;
```

Comment:
You're missing other forward decls for tags

Reply:
Not sure which, I think I have includes for all tags other than `Normalized`:
 - `Domain/FaceNormal.hpp` for `Tags::UnnormalizedFaceNormal`
 - `ScalarWave/Tags.hpp` for SW tags
...